apps/site-app/lang/en.json

```diff
@@ -435,6 +435,62 @@
     "defaultMessage": "The following invalid characters were found: {chars}",
     "description": "error message on invalid characters"
   },
+  "shared.widgets.balanceandprofit.balance": {
+    "defaultMessage": "Balance",
+    "description": "shared.widgets.balanceandprofit.balance: Title for widget Balance and profit"
+  },
+  "shared.widgets.balanceandprofit.cad": {
+    "defaultMessage": "$CAD",
+    "description": "shared.widgets.balanceandprofit.cad: Title for widget Balance and profit"
+  },
+  "shared.widgets.balanceandprofit.profit": {
+    "defaultMessage": "Profit",
```

Comment:
We should probably remove this file from git, and figure out a better way to track translations

Reply:
I am not sure what the other options are. The only other options I know of are just adding interfaces for translators.

Reply:
Neither - but there -must- be something?  Like, isn't this a problem that everyone everywhere has?  I did look into some BYO translation services, and there was one good one but required switching i18n library (just... no...)

Reply:
You are probably going to be mad at me for saying this again but: it was a bit better with the fixed ids.

Reply:
:-). There are pro's and con's of each approach.  Mostly I just didn't want to have to update the IDs to be correct after moving the component and didn't want to leave incorrect ids in there.

Translation is a problem I'll have another crack at soon.  We have to get the translations from shared/base merged into the final version at the very least.